src/devices/Card/CreditCard/ConvertionType.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Card.CreditCardProcessing
```

Comment:
this seems like a reduced copy of `ConversionType`

Reply:
Absolutely, forgot to remove this redundant file.